src/Sylius/Bundle/AdminBundle/Resources/views/Email/adminPasswordReset.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends '@SyliusAdmin/Email/layout.html.twig' %}
+
+{% block subject %}
+    {% include '@SyliusAdmin/Email/Blocks/AdminPasswordReset/_subject.html.twig' %}
```

Comment:
IMO extraction of this template is not needed, probably the email as a whole would be often overridden rather than only the subject 🖖